examples/stm32/l4/Makefile.include

```diff
@@ -0,0 +1,44 @@
+##
+## This file is part of the libopencm3 project.
+##
+## This library is free software: you can redistribute it and/or modify
+## it under the terms of the GNU Lesser General Public License as published by
+## the Free Software Foundation, either version 3 of the License, or
+## (at your option) any later version.
+##
+## This library is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+## GNU Lesser General Public License for more details.
+##
+## You should have received a copy of the GNU Lesser General Public License
+## along with this library.  If not, see <http://www.gnu.org/licenses/>.
+##
+
+# You should use linker script generation! Specify device!
+ifeq ($(DEVICE),)
+LIBNAME		= opencm3_stm32l4
+DEFS		+= -DSTM32L4
+
+FP_FLAGS	?= -mfloat-abi=hard -mfpu=fpv5-sp-d16
```

Comment:
where di you get that it's v5?

Reply:
You're right though, I have this as an unchecked in file locally, never committed when the "basics" example was added.